src/Files.App/MainWindow.xaml.cs

```diff
@@ -18,38 +21,31 @@ public sealed partial class MainWindow : WindowEx
 		private static MainWindow? _Instance;
 		public static MainWindow Instance => _Instance ??= new();
 
-		public IntPtr WindowHandle { get; }
+		public nint WindowHandle { get; }
 
-		private MainWindow()
+		public MainWindow()
 		{
-			WindowHandle = this.GetWindowHandle();
-
 			InitializeComponent();
 
-			EnsureEarlyWindow();
-		}
-
-		private void EnsureEarlyWindow()
-		{
-			// Set PersistenceId
-			PersistenceId = "FilesMainWindow";
-
-			// Set minimum sizes
-			MinHeight = 416;
-			MinWidth = 516;
+			WindowHandle = WinRT.Interop.WindowNative.GetWindowHandle(this);
+			ExtendsContentIntoTitleBar = true;
+			Title = "Files";
 
-			AppWindow.Title = "Files";
-			AppWindow.SetIcon(AppLifecycleHelper.AppIconPath);
-			AppWindow.TitleBar.ExtendsContentIntoTitleBar = true;
 			AppWindow.TitleBar.ButtonBackgroundColor = Colors.Transparent;
 			AppWindow.TitleBar.ButtonInactiveBackgroundColor = Colors.Transparent;
+			AppWindow.TitleBar.ButtonPressedBackgroundColor = Colors.Transparent;
+			AppWindow.TitleBar.ButtonHoverBackgroundColor = Colors.Transparent;
+			AppWindow.SetIcon(AppLifecycleHelper.AppIconPath);
+
+			MinHeight = 550;
+			MinWidth = 460;
```

Comment:
I'm not sure where the values 416 and 516 came from, but what's the reason for changing them?

Reply:
This was not intentional so I'll revert back soon.

I find the good min width and height for splash screen not to overwrap the splash screen text with progress ring.
But we can use the same values.

Reply:
Reverted